libcxx/test/std/ranges/range.adaptors/range.transform/iterator/deref.pass.cpp

```diff
@@ -37,7 +37,7 @@ int main(int, char**) {
     using View = std::ranges::transform_view<MoveOnlyView, PlusOneNoexcept>;
     View transformView(MoveOnlyView{buff}, PlusOneNoexcept{});
     assert(*transformView.begin() == 1);
-    LIBCPP_ASSERT_NOEXCEPT(*std::declval<std::ranges::iterator_t<View>>());
+    ASSERT_NOEXCEPT(*std::declval<std::ranges::iterator_t<View>>());
```

Comment:
The `operator*` previously provided a `noexcept` specifier as a libc++ extension. Following the adoption of this LWG issue, the `noexcept` specifier is now standard behavior.